tests/e2e/support/objects/app-files/resource/actions.ts

```diff
@@ -1346,3 +1346,12 @@ export const countNumberOfResourcesInThePage = ({ page }): Promise<number> => {
 export const expectPageNumberNotToBeVisible = async ({ page }): Promise<void> => {
   await expect(page.locator(filesPaginationNavSelector)).not.toBeVisible()
 }
+
+export interface checkBoxShouldBeChecked {
```

Comment:
I would expect better name
```suggestion
export interface checkBoxShouldBeChecked {
```